database/07_Edit_Post_Addition.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE post
+ADD COLUMN "editedTimestamp" TIMESTAMP WITH TIME ZONE;
```

Comment:
Postgres has a `timestamptz` shortcut that is recognized as a timestamp with time zone, just for your knowledge

Reply:
Ahhh I just saw it. Ill use that instead for this. Done